### PR TITLE
Droth 1693 tr height weight and bogie

### PR DESCRIPTION
**Pull request by marquesfm**

### CODE REVIEW

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TierekisteriPointConversionImporterSpec.scala [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+//    override lazy val municipalityDao: MunicipalityDao = mockMunicipalityDao
+//    override lazy val roadAddressService: RoadAddressesService = mockRoadAddressService
+//    override val tierekisteriClient: TierekisteriWeightLimitAssetClient = mockTierekisteriWeightLimitAssetClient
+//    override lazy val roadLinkService: RoadLinkService = mockRoadLinkService
+//    override lazy val vvhClient: VVHClient = mockVVHClient
+//  }
+
+//  class TestTotalWeightLimitImporter extends TotalWeightLimitImporter {
+//
+//  }
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TierekisteriPointConversionImporterSpec.scala]: check if could be removed

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TierekisteriPointConversionImporterSpec.scala [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+//    val endAddressMValue: Int = 3194
+//    val track: Track = Track.RightSide
+//
+//    val trAssetData = TierekisteriWeightLimitData(roadNumber, startRoadPartNumber, startRoadPartNumber, track, 1, 1 , Some(1000), None, None, None, None)
+//
+//    val trValue = tierekisteriPointConversionImporter.getValue(trAssetData)
+//
+//    trValue should be (1000)
+//  }
+}
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TierekisteriPointConversionImporterSpec.scala]: this test probably shouldn't be commented

File: UI/src/view/dynamicAssetForm.js [hunk 3]
```diff
@@ -879,15 +882,17 @@
...
-                    setValueFn({ properties: [] });
-                }
-                formGroup.find('.input-unit-combination').replaceWith(me.renderFormElements(asset, isReadOnly, sideCode, setValueFn, getValueFn, disabled));
+                  if(asset.value) {
+                      setValueFn(asset.value);
+                      formGroup.find('.input-unit-combination').replaceWith(me.renderFormElements(asset, isReadOnly, sideCode, setValueFn, getValueFn, disabled));
+                  }else {
+                      setValueFn({properties: []});
+                      formGroup.find('.input-unit-combination').replaceWith(me.renderFormElements(asset, isReadOnly, sideCode, setValueFn, getValueFn, disabled));
+                  }}
```
Comment (MAC-CGI) [inline UI/src/view/dynamicAssetForm.js]: could be simplified 
```
if(asset.value) {
   setValueFn(asset.value);	                       
}else {
   setValueFn({properties: []});
}
formGroup.find('.input-unit-combination').replaceWith(me.renderFormElements(asset, isReadOnly, sideCode, setValueFn, getValueFn, disabled));
}
```

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala [hunk 4]
```diff
@@ -304,27 +332,29 @@ object TierekisteriDataImporter {
...
+              importAssets(tierekisteriDataImporters(assetType))
           case "update" =>
             val lastExecutionDate = getDateFromArgs(args)
             if(assetType == "trafficVolume")
               println("The asset type trafficVolume doesn't support update operation.")
             else
-              updateAssets(tierekisteriDataImporters.get(assetType).get, lastExecutionDate)
+              updateAssets(tierekisteriDataImporters(assetType), lastExecutionDate)
+          case "converter" =>
+              importAssets(tierekisteriDataConverter(assetType))
```
Comment (moraispedro) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala]: Wouldn't it be better to add a default case?

File: UI/src/view/dynamicAssetForm.js (hunk 3 above)
Comment (oskarmalmstrom) [reply UI/src/view/dynamicAssetForm.js]: maybe even more: 

```
setValueFn(asset.value || {properties: []} );
formGroup.find('.input-unit-combination').replaceWith(me.renderFormElements(asset, isReadOnly, sideCode, setValueFn, getValueFn, disabled));

```

File: UI/src/assetTypeConfiguration.js [hunk 5]
```diff
@@ -137,7 +137,7 @@
         title: 'Suurin sallittu telimassa',
         newTitle: 'Uusi suurin sallittu telimassa',
         className: 'bogie-weight-limit',
-        unit: 'kg',
+        unit: 'Kg',
```
Comment (oskarmalmstrom) [inline UI/src/assetTypeConfiguration.js]: should be 'kg'

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala (hunk 4 above)
Comment (oskarmalmstrom) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala]: Will probably throw an exception if no match. but we could print something like: 
`case _ => println("Usage: import | update | converter")`

Comment (oskarmalmstrom) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala]: on the other hand, on the dropdown we only have those options so...

**Pull request merged**